task/04-arrays-tasks.js

```diff
@@ -337,7 +384,11 @@ function sortDigitNamesByNumericOrder(arr) {
  *   [ 1, 10, 100, 1000 ]  => 1111
  */
 function getItemsSum(arr) {
-   throw new Error('Not implemented');
+   if(arr.length>0){
```

Comment:
it is necessary to observe indents.
Example: "if (a > b) {"
https://javascript.info/coding-style